pkgs/by-name/ty/typr/package.nix

```
@@ -0,0 +1,34 @@
+{
+  lib,
+  python3,
+  python3Packages,
+  fetchFromGitHub,
+}:
+
+python3.pkgs.buildPythonPackage rec {
+  pname = "typr";
+  rev = "ca2b1dd5d3796904cc83830d245ebe0978ed87c0";
+  version = "1.0.1.20";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "DriftingOtter";
+    repo = "Typr";
+    rev = "${rev}";
```

don't overthink this
```suggestion
  version = "1.0.1.20";
  pyproject = true;

  src = fetchFromGitHub {
    owner = "DriftingOtter";
    repo = "Typr";
    rev = "ca2b1dd5d3796904cc83830d245ebe0978ed87c0";
```